resources/views/mail/ticket/status-updated.blade.php

```diff
@@ -0,0 +1,27 @@
+@php
+    $ticketableType = $ticketable instanceof \App\Models\Achievement ? 'achievement' : 'leaderboard';
+@endphp
+
+<x-mail::message>
+Hello {{ $ticket->reporter->display_name }}!
+
+@if ($ticket->reporter->is($updatedBy))
+The ticket you opened for the following {{ $ticketableType }} had its status changed to *{{ $newStatus }}*.
+@else
+The ticket you opened for the following {{ $ticketableType }} had its status changed to *{{ $newStatus }}* by *{{ $updatedBy->display_name }}*.
```

Comment:
It feels weird to have this in the summary and body of an email:

<img width="523" height="226" alt="Image" src="https://github.com/user-attachments/assets/5ae81d42-909b-44c6-a074-31551ae3c822" />

Reply:
Updated this email content in latest:
<img width="550" height="253" alt="Screenshot 2025-09-01 at 2 34 43 PM" src="https://github.com/user-attachments/assets/8a8c902f-76c4-4454-b7c7-4147cdd96c52" />